.pre-commit-config.yaml

```diff
@@ -1,25 +1,33 @@
 repos:
+
+
 - repo: https://github.com/psf/black
   rev: 22.3.0
   hooks:
   - id: black
-    exclude: "^({{cookiecutter.project_slug}}/)"
```

Comment:
Hmmm... ``{{cookiecutter.project_slug}}`` - maybe you should think about substituting it for your actual project directory (if you need the "exclude" statement of course, otherwise well done! =) )